### DISCUSSION TITLE
Nuxt 3 compatible

### DISCUSSION THREAD

**Discussion by Gruce** [tauri-apps/tauri#4391] [Q&A] [answered]
### Hello
I am just wondering if there is any plan in the future to make Tauri compatible with Nuxt 3. It would be great together.

Thank you.

**Comment by FabianLars** [comment, marked as answer]
Tauri should already be compatible with it. As i always like to say it "Tauri doesn't care about your frontend". The only thing here is that you need to disable nuxt's SSR mode to get a serverless (=client side) app because there is no server or nodejs runtime in a Tauri app. Btw they are also working on an SSG mode :shrug: 

Alternatively you could [sidecar](https://tauri.app/v1/guides/building/sidecar/) a nodejs+nuxt server with your tauri app, but yeah that's pretty much an anti-pattern :sweat_smile:

**Reply by Lyokolux** [reply]
To put some update to new readers, Elk is a Nuxt 3 backed app that has a native implementation with tauri: https://github.com/elk-zone/elk-native

**Comment by HuakunShen** [comment]
It would be nice if Nuxt 3 template can be integrated into the scaffold.

**Reply by FabianLars** [reply]
We removed all SSR-first frameworks from create-tauri-app in [v3](https://tauri.app/blog/2023/03/01/create-tauri-app-version-3-released) so this is highly unlikely. What would be possible is a getting started guide in the docs if a community member writes one :)

**Reply by simonhyll** [reply]
Here's a video I made on using Nuxt 3
https://youtu.be/MOnf_kGI6L0

**Reply by HuakunShen** [reply]
> We removed all SSR-first frameworks from create-tauri-app in [v3](https://tauri.app/blog/2023/03/01/create-tauri-app-version-3-released) so this is highly unlikely. What would be possible is a getting started guide in the docs if a community member writes one :)

Yeah I noticed that sveltekit template is no longer there, I have to set it up myself.

**Comment by simonhyll** [comment]
Here's a video on how to do it. There's currently no intention of adding Nuxt to the templates. I even made one a while back and made a PR for it, but for v3 we chose to remove templates rather than add more. If your favorite framework isn't in there it's probably because we intentionally left it out, not because we don't recommend it but because we have a limited nr of developers capable of maintaining templates.
https://youtu.be/MOnf_kGI6L0

**Comment by HuakunShen** [comment]
I made a template repo https://github.com/HuakunShen/tauri-nuxt-template and an [automation script](https://github.com/HuakunShen/tauri-nuxt-template#auto-script-for-generating-a-template) to generate everything.
